### PR TITLE
New questionnaires

### PR DESCRIPTION
**Pull request by yatharthranjan**
- Update pulse ox to slider input
- Add exacerbation diary to ILD

### CODE REVIEW

File: questionnaires/exacerbation_diary_ild/exacerbation_diary_ild_armt.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
+[
+    {
+        "field_name": "how_likely_do_you_feel_it_46e8ec_ild",
```
Comment (mpgxvii) [inline questionnaires/exacerbation_diary_ild/exacerbation_diary_ild_armt.json:3]: Is the `46e8ec` here intentional?

**Review by mpgxvii — APPROVED** [review]
LGTM

File: questionnaires/exacerbation_diary_ild/exacerbation_diary_ild_armt.json (hunk 1 above)
Comment (afolarin) [reply questionnaires/exacerbation_diary_ild/exacerbation_diary_ild_armt.json:3]: Hmm... I know what happened. I copied the original Exacerbation Diary, but redcap will append a disambiguation string to the variable names. We definitely should refactor these, I will update the form in REDCap to rename more sensibly.

Comment (afolarin) [reply questionnaires/exacerbation_diary_ild/exacerbation_diary_ild_armt.json:3]: Although @yatharthranjan  I note the original Exacerbation diary has some junk string at the end of the var name (should this be cleaned up toop?) https://github.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/blob/master/questionnaires/exacerbation_diary/exacerbation_diary_armt.json

Comment (afolarin) [reply questionnaires/exacerbation_diary_ild/exacerbation_diary_ild_armt.json:3]: @mpgxvii I've updated this one to remove the junk string. Though the var name is over 26chars it is ok (but @yatharthranjan if you want to shorten both in the questionnaires then that is also fine something like `rate_exacerbation_copd` and `rate_exacerbation_ild` would probably be ok

Comment (yatharthranjan) [reply questionnaires/exacerbation_diary_ild/exacerbation_diary_ild_armt.json:3]: yeah, I duplicated that questionnaire once and then deleted the old one, I think that is how the random string got in. I will update it too.

Comment (yatharthranjan) [reply questionnaires/exacerbation_diary_ild/exacerbation_diary_ild_armt.json:3]: Actually it was because ERS has the same field name

Comment (yatharthranjan) [reply questionnaires/exacerbation_diary_ild/exacerbation_diary_ild_armt.json]: Ok updated now

**Review by afolarin — APPROVED** [review]
LGTM

**Pull request merged**